pkg/inventory/container/ocp/reconciler.go

```diff
@@ -412,7 +412,7 @@ func (r *ModelEvent) Apply(rl *Reconciler) (err error) {
 			rl.log.V(3).Info(
 				"model created.",
 				ref.ToKind(r.model),
-				r.model.String())
+				libmodel.Describe(r.model))
```

Comment:
String() no longer part of the `Model` interface.  Should be using `Describe()` anyway .. much better.